start_esp/start_esp.py

```diff
@@ -939,6 +940,11 @@ def make_argparser():
     parser.add_argument('--jwks_cache_duration_in_s', default=None, type=int, help='''
         Specify JWT public key cache duration in seconds. Default is 5 minutes.''')
 
+    parser.add_argument('--redirect_authorization_url', action='store_true',
+                        help='''If specified, authentication failed requests will be redirected
```

Comment:
How about `If set to true, authentication failed requests will be redirected to the URL specified by the `authorizationUrl` field in OpenAPI spec. The default is false.`?

Reply:
Done